lib/workers/branch/commit.ts

```diff
@@ -46,11 +48,16 @@ export function commitFilesToBranch(
     );
     throw new Error(CONFIG_SECRETS_EXPOSED);
   }
-  // API will know whether to create new branch or not
-  return commitFiles({
+
+  const commitConfig: CommitFilesConfig = {
     branchName: config.branchName,
     files: updatedFiles,
     message: config.commitMessage,
     force: !!config.forceCommit,
-  });
+  };
+
+  // API will know whether to create new branch or not
+  return config.platformCommit && platform.commitFiles
```

Comment:
Should we warn for unsupported platforms?

I would also extract this code for deduplication.